MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,376 @@
+
+## Display attribute
+
+<table>
+<tr>
+<th>Property</th>
+<th>Details</th>
+</tr>
+<tr>
+<td>
+{{'[Name](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.name?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the label text.
+</td>
+</tr>
+<tr>
+<td>
+{{'[GroupName](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.groupname?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the group name which groups the fields in the data form.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ShortName](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.shortname?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the label text. It takes higher priority than Name.
+</td>
+</tr>
+<tr>
+<td>
+{{'[AutoGenerateField](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.autogeneratefield?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies whether the field should be auto generated or not. 
+</td>
+</tr>
+<tr>
+<td>
+{{'[ResourceType](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.resourcetype?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the Resources File (.Resx) which is used to localize the Display attribute of `Name`, `ShortName`, `GroupName`, `Prompt` and `ItemsSource` values.
+</td>
+</tr>
+<tr>
+<td>
+{{'[Prompt](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.prompt?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies watermark text for the editor.
+</td>
+</tr>
+<tr>
+<td>
+{{'[Order](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.order?view=net-7.0)' | markdownify }}
+</td>
+<td>
+Specifies the order of field in the data form.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight C# attribute}
+
+    [Display(Name = "First Name", GroupName = "Name", Prompt="Enter your name")]
+    public string FirstName { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validation attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+{{'[MinLength](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.minlengthattribute?view=net-7.0)'| markdownify }}
+</td>
+<td>
+Specifies the required minimum input string length.
+</td>
+</tr>
+<tr>
+<td>
+{{'[MaxLength](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.maxlengthattribute?view=net-7.0)'| markdownify }}
+
+</td>
+<td>
+Specifies the required maximum input string length.
+</td>
+</tr>
+<tr>
+<td>
+{{'[Required](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=net-7.0)'| markdownify }}
+</td>
+<td>
+Specifies that a data field value is required. 
```

Comment:
Specifies that the data field value is required.

Reply:
the word added